api/services/ocean/rightsizing/responses/oceanRightsizingRule.yaml

```diff
@@ -36,4 +36,8 @@ properties:
   recommendationApplicationHPA:
     $ref: "../schemas/oceanRightsizingRuleRecommendationApplicationHPA.yaml"
   autoApplyDefinition:
-    $ref: "../schemas/oceanRightsizingRuleAutoApplyDefinition.yaml"
\ No newline at end of file
+    $ref: "../schemas/oceanRightsizingRuleAutoApplyDefinition.yaml"
+  downsideOnly:
+    type: boolean
+    description: When set to 'true', only down scale recommendation application will take place.
```

Comment:
`down scale` -> `downscale`

wdyt about `When set to ‘true’, only downscale recommendations will be applied`